### ISSUE TITLE
No Output Unless Parser Option is Specified

### ISSUE THREAD

**Issue by redxtech** [vuejs/eslint-plugin-vue#387]
**Tell us about your environment**

* **ESLint Version:** 4.17.0
* **eslint-plugin-vue Version:** 4.2.2
* **Node Version:** 8.0.0

**Please show your full configuration:**

```js
module.exports = {
  root: true,
  parserOptions: {
    parser: 'babel-eslint',
    sourceType: 'module'
  },
  env: {
    browser: true
  },
  plugins: [
    'vue',
    'html'
  ],
  extends: [
    'standard',
    'plugin:vue/recommended'
  ]
}
```

**What did you do? Please include the actual source code causing the issue.**

```vue
<template>
    <div :id="`plyr-container-${this.idNumber}`">
        <slot/>
    </div>
</template>

<script>
  import plyr from 'plyr'
  import 'plyr/dist/plyr.css'

  export default {
    name: 'plyr',
    props: {
      /** Options object for plyr config. */
      options: {
        type: Object,
        required: false,
        default () { return {} }
      }
    },
    data () {
      return {
        player: {}
      }
    },
    mounted () {
      this.player = plyr.setup(document.getElementById(`plyr-container-${this.idNumber}`),
        this.options)[0]
    },
    beforeDestroy () {
      this.player.destroy()
    },
    computed: {
      idNumber () {
        return Math.floor(Math.random() * (100000 - 1)) + 1
      }
    }
  }
</script>
```

**What did you expect to happen?**
I expected the plugin to work as eslint plugins do, and give me a list of the pieces of code that do not conform.

**What actually happened? Please include the actual, raw output from ESLint.**
Nothing. I just run `eslint --ext .js,.vue src` through yarn as a script, and I get no output.
[Here's a screenshot](https://i.gyazo.com/3f6cc3dcbc3cb943476027973e7014ea.png).
The only time I get an output is when I move `parser: 'babel-eslint'` to outside the `parserOptions` property. The output I get there is a bunch of errors related to use the latest vue eslint parser (to be expected) and some messages telling me the actual problems (as it should normally).
[Here's a screenshot](https://i.gyazo.com/87c02c3466dd8b1597e970f833afea27.png).

Is there anything I'm doing wrong/that I can do to fix this ?
Thanks \:)

**Comment by michalsnik** [comment]
It's probably because of `html` plugin @redxtech, remove it as we no longer depend on that and this plugin is not compatible with `eslint-plugin-html` anymore.

**Comment by redxtech** [comment]
That appears to be a fix, thanks.
